ui-v2/app/styles/components/tag.scss

```diff
@@ -1,6 +1,6 @@
 %tag,
 %pill {
-  display: inline-block;
-  padding: 1px 5px;
   border-radius: $radius-small;
+  display: inline-block;
+  padding: 3px 6px;
```

Comment:
This has knocked out the 'vertical centralling' of the tags within the row, notice the space below the tag is now larger than the space above the tag.

I'd probably revert this and keep this to just color changes for the minute. Or at least revert the top and bottom padding change.

👍 I think you are right though, they do need more space, could probably do with looking at how things are positioned within `td`'s instead of using padding. Looking at the tags more closely in these screenshots the text probably needs raising a nudge also.

Before:

![screen_shot_2018-07-20_at_10_17_58](https://user-images.githubusercontent.com/554604/42994802-0ce7af8e-8c07-11e8-9f4c-83341623457b.png)

After:

![screen_shot_2018-07-20_at_10_18_17](https://user-images.githubusercontent.com/554604/42994808-119d39a4-8c07-11e8-87d2-48cfd04e2c88.png)

Reply:
I agree the tags and row alignment need some work, but can definitely "table" this for later. 

See what I did there? 😜